src/lib/ibcclient.ts

```diff
@@ -82,6 +83,20 @@ import {
   toIntHeight,
 } from './utils';
 
+function formatMessage<T>(
+  message: T,
+  mutateFn: (
+    deepClonedMessage: T,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    encode: (data: Uint8Array) => any
```

Comment:
I would actually remove this from here, and leave it inside the messages (maybe we want different ways to encode different binary fields).

Reply:
addressed in https://github.com/confio/ts-relayer/pull/159/commits/bf99247f7cf609125b03c73919a9760f41cf8cb0